### DISCUSSION TITLE
2a) Time stepping - Initial Conditions

### DISCUSSION THREAD

**Discussion by jack-slingsby** [wenceorg/comp4187#101] [Q&A] [answered]
Hi,
I was wondering if there are any inital conditions we should use when timestepping in Q2a?

Thanks

**Comment by annereinarz** [comment, marked as answer]
The only thing to watch out for is that the initial condition fulfills the boundary conditions. The total number of Euler steps that are needed to reach the steady state will depend on the choice, but you can't really do anything "wrong". Zero is a common choice.

**Reply by jack-slingsby** [reply]
Awesome, Thank you!
